job-dsls/src/main/resources/job-scripts/prod_create_release_trackers.jenkinsfile

```
@@ -0,0 +1,42 @@
+pipeline {
+    agent {
+        label 'kie-rhel7 && !master'
+    }
+
+    stages {
+        stage('Clean workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+
+        stage('Print variables') {
+            steps {
+                println "[INFO] VERSION: ${VERSION}"
+                println "[INFO] TEMPLATE: ${TEMPLATE}"
+            }
+        }
+
+        stage('Clone the repository') {
+            steps {
+                sh "git clone ${INTEGRATION_PLATFORM_TOOLING_REPOSITORY}"
```

let's suppose the repository is renamed, `dir('integration-platform-tooling')` would fail. So I suggest to force the output directory.

```suggestion
                sh "git clone ${INTEGRATION_PLATFORM_TOOLING_REPOSITORY} integration-platform-tooling"
```